modules/apps/commerce/commerce-product-type-grouped-web/src/main/resources/META-INF/resources/render/view.jsp

```diff
@@ -35,7 +35,7 @@ String productContentAuthToken = AuthTokenUtil.getToken(request, plid, CPPortlet
 		<div class="product-detail-header">
 			<div class="col-lg-6 col-md-7">
 				<div class="row">
-					<div class="col-lg-2 col-md-3 col-xs-2">
+					<div class="col-lg-2 col-md-3 col-2">
```

Comment:
should we use the `clay:col`, `clay:row` tags instead?

Reply:
**yes**!! :-D
Could we do it after fixing the compatibility task?